src/components/exchange/exchange.css

```diff
@@ -0,0 +1,8 @@
+.exchange {
+  display: flex;
```

Comment:
delete?  there is only one child element, so I don't think `display: flex` does anything in this case